Community Balance Patch/Balance Changes/Units/MountedUnits.sql

```diff
@@ -30,8 +30,8 @@
 	UPDATE Units SET Moves = '5' WHERE Type = 'UNIT_MOUNTED_BOWMAN' AND EXISTS (SELECT * FROM COMMUNITY WHERE Type='COMMUNITY_CORE_BALANCE_UNITS' AND Value= 1 );
 	UPDATE Units SET Moves = '5' WHERE Type = 'UNIT_CUIRASSIER' AND EXISTS (SELECT * FROM COMMUNITY WHERE Type='COMMUNITY_CORE_BALANCE_UNITS' AND Value= 1 );
 	UPDATE Units SET Moves = '5' WHERE Type = 'UNIT_INDIAN_WARELEPHANT' AND EXISTS (SELECT * FROM COMMUNITY WHERE Type='COMMUNITY_CORE_BALANCE_UNITS' AND Value= 1 );
-	UPDATE Units SET Moves = '5' WHERE Type = 'UNIT_HELICOPTER_GUNSHIP' AND EXISTS (SELECT * FROM COMMUNITY WHERE Type='COMMUNITY_CORE_BALANCE_UNITS' AND Value= 1 );
 	UPDATE Units SET Moves = '5' WHERE Type = 'UNIT_CAVALRY' AND EXISTS (SELECT * FROM COMMUNITY WHERE Type='COMMUNITY_CORE_BALANCE_UNITS' AND Value= 1 );
+	UPDATE Units SET Moves = '5' WHERE Type = 'UNIT_LIGHT_TANK' AND EXISTS (SELECT * FROM COMMUNITY WHERE Type='COMMUNITY_CORE_BALANCE_UNITS' AND Value= 1 );
```

Comment:
For the light tank is "UNIT_ANTI_TANK_GUN", should remove this line
And also UNIT_ANTI_TANK_GUN is present twice with the movement update, I've changed them in both and there's no change in game (still shows 4 movement), @LoneGazebo hope can fix this, and I think this pull request should be closed...

Reply:
So it did work after all with the second change in the file that had to be made for "UNIT_ANTI_TANK_GUN" (I don't now why didn't work previously), I made a pull req just with this...